dashboard/modules/snapshot/tests/test_snapshot.py

```diff
@@ -162,7 +162,7 @@ def my_func_nondetached(request):
     assert entry_deleted["name"] == "my_func_deleted"
     assert entry_deleted["version"] == "v1"
     assert entry_deleted["namespace"] == "serve"
-    assert entry_deleted["httpRoute"] == "/my_func_deleted"
+    assert entry_deleted["httpRoute"] is None
```

Comment:
When a deployment is deleted, its route gets removed. Previously this did not matter because we put `route_prefix or`/{name}` as the value for `httpRoute`, leading this deployment to still have a route listed in the snapshot (although it doesn't have one attached).